LabTools/IO/IOTool.py

```diff
@@ -395,6 +398,32 @@ def get_debug_setting(**kwargs):
 def get_interface_setting(**kwargs):
     return get_config_setting(GPIB_INTF_ID, **kwargs)
 
+def get_visa_backend_setting(**kwargs):
+    """
+        This function returns the backend which will be input into visa.ResourceManager(backend)
+        
+        @ni uses national instruments backend (proprietary)
+        @py uses pure python pyvisa backend (open source)
+        
+        NOTE: py requires other packages to be installed, such as linux-gpib, pyusb, pyserial, etc, etc
```

Comment:
visa backend as well, you have to download a 600+ MB driver from their website before you can use their GBIP to USB, they were probably already installed on the machines you tested (and I am not sure how it works on linux)